### PR TITLE
Update Transactions Page to reflect new mockup 

### LINKED ISSUE #651: Transactions page update
Currently, this is what the transactions page look like.
image

It should be updated to.
image

And the transactions details hash to this
image

### PR DESCRIPTION
**Pull request by devchoplife**
Closes #651 
This PR implements the following:
- Adds year to the time format on transactions row
- Adds DurationAgo function that gets exact time since a txn has been made 
- Adds the time ago for txn on transactions row

This PR will not address the trnansaction details hash as the page is still undergoing changes

### CODE REVIEW

**Comment by dreacot** [comment]
Screenshot from 2021-10-18 18-45-13

@devchoplife we have December and August showing 1 year ago, is that intended, or shouldn't the month ago also be added?

File: ui/page/components/components.go [hunk 1]
```diff
@@ -165,6 +166,105 @@ func TransactionTitleIcon(l *load.Load, wal *dcrlibwallet.Wallet, tx *dcrlibwall
...
 }
 
+func DurationAgo(timestamp int64) string {
+	//Convert timestamp to date in string format (yyyy:mm:dd hr:m:s +0000 UTC)
+	currentTimestamp := time.Now().UTC().String()
+	txnTimestamp := time.Unix(timestamp, 0).UTC().String()
+	//Split the date so we can sepparate into date and time for current time and time of txn
+	currentTimeSplit := strings.Split(currentTimestamp, " ")
+	txnTimeSplit := strings.Split(txnTimestamp, " ")
+	//Split current date and time, and  txn date and time then store in variables
```
Comment (dreacot) [inline ui/page/components/components.go]:
```suggestion
	// Convert timestamp to date in string format (yyyy:mm:dd hr:m:s +0000 UTC)
	currentTimestamp := time.Now().UTC().String()
	txnTimestamp := time.Unix(timestamp, 0).UTC().String()
	// Split the date so we can sepparate into date and time for current time and time of txn
	currentTimeSplit := strings.Split(currentTimestamp, " ")
	txnTimeSplit := strings.Split(txnTimestamp, " ")
	// Split current date and time, and  txn date and time then store in variables
```
Leave a space before typing the comments

**Comment by devchoplife** [comment]
> Screenshot from 2021-10-18 18-45-13
> 
> @devchoplife we have December and August showing 1 year ago, is that intended, or shouldn't the month ago also be added?

Not intended 
A condition is missing 
Working on that

File: ui/page/components/components.go [hunk 2]
```diff
@@ -352,16 +468,28 @@ func TxConfirmations(l *load.Load, transaction dcrlibwallet.Transaction) int32 {
...
+
+	if strconv.Itoa(currentTime.Year()) == strconv.Itoa(utcTime.Year()) && currentTime.Month().String() == utcTime.Month().String() {
+		if strconv.Itoa(currentTime.Day()) == strconv.Itoa(utcTime.Day()) {
+			if strconv.Itoa(currentTime.Hour()) == strconv.Itoa(utcTime.Hour()) {
+				return TimeAgo(timestamp)
+			}
+
+			return TimeAgo(timestamp)
+		} else if currentTime.Day()-1 == utcTime.Day() {
+			ans := "Yesterday"
```
Comment (dreacot) [inline ui/page/components/components.go]: what's `ans` ? rename `ans` to something more appropriate

**Review by dreacot — CHANGES_REQUESTED** [review]

**Review by dreacot — COMMENTED** [review]
also the transaction hash hasn't been truncated as requested

**Review by dreacot — CHANGES_REQUESTED** [review]
- 2 weeks ago16days, should be 2 weeks 16 days ago
- after having 2weeksago16days, the next line decreases and says 2 weeks, that's not right
- Last month should read 1 month ago
Screenshot from 2021-10-20 19-57-53

- Last year should say 1 year ago
Screenshot from 2021-10-20 20-01-29

**Review by dreacot — APPROVED** [review]

**Pull request merged**